state/state.go

```diff
@@ -1009,6 +1010,7 @@ func (s *State) Init() {
 	s.msgQueue = make(chan interfaces.IMsg, 50)                             //queue of Follower messages
 	s.prioritizedMsgQueue = make(chan interfaces.IMsg, 50)                  //a prioritized queue of Follower messages (from mmr.go)
 	s.MissingEntries = make(chan *MissingEntry, constants.INMSGQUEUE_HIGH)  //Entries I discover are missing from the database
+	s.dataQueue = NewInMsgQueue(constants.INMSGQUEUE_HIGH)                  //incoming requests for missing data
```

Comment:
INMSGQUEUE_HIGH is a big size? Processing should be pretty fast. Do you have reason to thing it needs to be big ? The channel for missing_messages is like 20 messages deep.

Reply:
no reason - I can set it to something reasonable like 20